lib/rubber/cloud/yaml.rb

```diff
@@ -0,0 +1,92 @@
+require 'rubber/cloud/generic'
+require 'yaml/store'
+
+module Rubber
+  module Cloud
+    # YAML file containing a set of servers available for use.
+    class YAML < Base
+      PROVIDER = 'YAML'.freeze
+      AVAILABLE = 'available'.freeze
+      ACTIVE = 'running'.freeze
+      STOPPED = 'stopped'.freeze
+
+      def database_file
+        ENV["YAML_DATABASE"]
```

Comment:
yes the user is expected to provide the full path. not a fan of standardizing this atm.